_posts/en/newsletters/2020-11-11-newsletter.md

```diff
@@ -0,0 +1,145 @@
+---
+title: 'Bitcoin Optech Newsletter #123'
+permalink: /en/newsletters/2020/11/11/
+name: 2020-11-11-newsletter
+slug: 2020-11-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter shares the announcement of a marketplace for
+incoming LN channels. Also included are our regular sections with summaries of
+a Bitcoin Core PR Review Club meeting and notable changes to popular Bitcoin
+infrastructure software.
+
+## Action items
+
+*None this week.*
+
+## News
+
+- **Incoming channel marketplace:** Olaoluwa Osuntokun [announced][pool
+  announce] a new marketplace for buying incoming LN channels.  Users and
+  merchants need channels with incoming capacity in order to be able to
+  quickly receive funds over LN.  Some existing node operators already
+  provide incoming channels, either for free or as a paid service, but
+  the Lightning Pool marketplace hopes to make this service more
+  standardized and competitive.  The initial focus is a contract for
+  [highly ranked][node rankings] node operators to provide an incoming
+  channel for a period of 2,016 blocks (about two weeks).  Additional
+  contract lengths and other features are planned for the future.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Add MuHash3072 implementation][review club #19055-2] is a PR ([#19055][Bitcoin
+Core #19055]) by Fabian Jahr (incorporating code originally written by Pieter
+Wuille) that implements the [MuHash algorithm][muhash mailing list] in C++.
+MuHash is a rolling hash algorithm that can be used to calculate the hash
+digest of a set of objects and efficiently update that digest when items are
+added to or removed from the set. The algorithm could be used to calculate a
+digest of the complete UTXO set, which would be useful for database consistency
+checks or for a fast sync method such as [AssumeUTXO][topic assumeutxo].
```

Comment:
Recommending s/AssumeUTXO/assumeUTXO

Reply:
We seem to be inconsistent here, using both assumeutxo (https://bitcoinops.org/en/newsletters/2019/06/12/#assume-utxo-demo) and AssumeUTXO (https://bitcoinops.org/en/newsletters/2019/12/28/#assumeutxo). Perhaps one to add to the style guide?

For now, I've changed this to assumeutxo.